index.html

```diff
@@ -0,0 +1,255 @@
+                  <span><img class="catchphrase__yyy" src="http://blog.coreknowledge.org/wp-content/uploads/2009/02/gusteau.jpg" alt=""></span>
+
+                  <div class="side-by-side">
+                    <h4 class="catchphrase__character">Chef Gusteau</h4>
+                    <p class="catchphrase__show">Ratatouille</p>
+                  </div>
+                </figcaption>
+              </figure>
+            </div>
+
+            <div class="hero__item hero__text">
+              <figure class="quote">
+                <div class="quote__card">
+                  <p class="quote__text">
+                    I ain't afraid to die anymore. I'd done it already.
+                  </p>
+                </div>
+                <figcaption class="catchphrase__zzz">
+                  <span><img class="catchphrase__yyy" src = "http://images6.fanpop.com/image/photos/39200000/The-Revenant-Leonardo-DiCaprio-as-Hugh-Glass-the-revenant-39221869-600-695.jpg" alt=""></span>
+
+                  <div class="side-by-side">
+                    <h4 class="catchphrase__character">"Hugh Glass"</h4>
+                    <p class="catchphrase__show">The Revenant</p>
+                  </div>
+                </figcaption>
+              </figure>
+            </div>
+
+            <div class="hero__item hero__text">
+              <figure class="quote">
+                <div class="quote__card">
+                  <p class="quote__text">
+                    Memory is a strange thing.
+                  </p>
+                </div>
+                <figcaption class="catchphrase__zzz">
+                  <span><img class="catchphrase__yyy" src="https://comicsgrinder.files.wordpress.com/2016/11/amy-adams-louise-banks-arrival.jpg" alt=""></span>
+
+                  <div class="side-by-side">
+                    <h4 class="catchphrase__character">Dr. Louise Banks</h4>
+                    <p class="catchphrase__show">The Arrival</p>
+                  </div>
+                </figcaption>
+              </figure>
+            </div>
+
+            <div class="hero__item hero__text">
+              <figure class="quote">
+                <div class="quote__card">
+                  <p class="quote__text">
+                    Not quite my tempo.
+                  </p>
+            </div>
+
+                <figcaption class="catchphrase__zzz">
+                  <span><img class="catchphrase__yyy" src="http://www.rig-talk.com/forum/download/file.php?id=47176&mode=view" alt=""></span>
+
+                  <div class="side-by-side">
+                    <h4 class="catchphrase__character">Terrence Fletcher</h4>
+                    <p class="catchphrase__show">Whiplash</p>
+                  </div>
+                </figcaption>
+              </figure>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+      <div class="section__photos" id="photos">
+        <div class="box">
+          <div class="hero">
```

Comment:
This section shouldn't need to reuse the `hero` class. Maybe see what classes you haven't used in the CSS file?